### ISSUE TITLE
Nominate/Validate buttons do not update

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/apps#997]
Goal:

Make things actually reflect what is on-chain.

Background:

Simple test. Do actions, see if the display matches up with that in a new tab. It does not quite - https://youtu.be/lkJmZR_m9Mc

I think we are pulling something really wrong here - the subscriptions should be updating per account. (I'm _assuming_ that is what goes wrong here, some sub is not updating or correctly queried, have no real idea atm without looking what info is not updating)

Actions:

- Investigate
- Fix :)

EDIT: What I think goes wrong - the global list (for some reason) is not updated and we use that. (Which is anyway probably not such a cool idea anyway and we should get rid or it)

So the problem seems to lie here - https://github.com/polkadot-js/apps/blob/master/packages/app-staking/src/Account/index.tsx#L355-L357

Basically, looking at https://polkadot.js.org/api/METHODS_STORAGE.html#staking -

- To determine is the guy is a wannabe validator: `validators(AccountId): (ValidatorPrefs, Linkage<AccountId>)`
- To determine if the guy is a wannabe nominator: 

Pull both those in the the `Account` component: ` nominators(AccountId): (Vec<AccountId>, Linkage<AccountId>)`

(Both via the stash)
